ablator/main/model/wrapper.py

```diff
@@ -821,18 +829,18 @@ def validation_loop(
         for i, batch in enumerate(dataloader):
             with torch.no_grad():
                 outputs, loss = self._model_step(model=model, batch=batch)
+                val_metrics = {}
                 if outputs is not None:
                     aux_metrics = self.aux_metrics(outputs)
                     metrics.append_batch(tag=tag, **outputs)
-
-                val_metrics = {}
+                    if aux_metrics is not None:
```

Comment:
Did we add a test for this change? @h21565897

Reply:
Already: https://github.com/fostiropoulos/ablator/blob/07e43eed45bb12d683471856c25b907af9da10d6/tests/main/test_wrapper.py#L405